src/cmdlib.sh

```diff
@@ -73,6 +73,10 @@ prepare_build() {
     # didn't create this in `init`.
     mkdir -p ${workdir}/tmp
 
+    # Needs to be absolute for rpm-ostree today
+    export changed_stamp=$(pwd)/tmp/treecompose.changed
+    rm -f ${changed_stamp}
```

Comment:
Minor: let's move the `rm -f` to right before `compose tree` down there to make it more obvious we're doing this right?

Reply:
Sure, done :arrow_down: